app/models/project.rb

```diff
@@ -13,6 +13,7 @@ class Project < ActiveRecord::Base
 
   EXPERT_ROLES = [:expert, :owner]
 
+  belongs_to :default_tutorial, class_name: "Tutorial"
```

Comment:
should this association be the other way around? `Project has_one :default_tutorial` and a `Tutorial belongs_to :project`?